docs/tutorials/transaction-tutorial.md

```diff
@@ -94,18 +114,33 @@ See our [TypeORM guide](./using-typeorm.md) for more information.
 <TabItem value="prisma" label="Prisma">
 
 ```javascript
+//Model specified in prisma/schema.prisma:
+//
+//model GreetingRecord {
+//  @@map("greetings") 
+//  greeting_id Int @id @default(autoincrement()) //Note: Prisma requires at least one primary key
+//  name String
+//  note String
+//}
+
 import { PrismaClient } from "@prisma/client"; // Use the generated Prisma client
 
 export class Greetings {
+  //...
   @Transaction()
-  static async insertGreeting(ctxt: TransactionContext<PrismaClient>, name: string, note: string)  {
-    await ctxt.client.greetings.create({
+  static async insertGreeting(ctxt: TransactionContext<PrismaClient>, name: string, note: string) {
+    await ctxt.client.greetingRecord.create({
       data: {
         name: name,
-        note: note,
+        note: note
       },
     });
   }
+
+  @Transaction({ readOnly:true })
+  static async getGreetings(ctxt: TransactionContext<PrismaClient>): Promise<{greeting_id: number, name:string, note:string}[]> {
```

Comment:
You can directly use the defined GreetingRecord from PrismaClient. Example: https://github.com/dbos-inc/dbos-demo-apps/blob/main/bank/bank-backend/src/workflows/txnhistory.workflows.ts#L16

```ts
import { AccountInfo, PrismaClient, TransactionHistory } from "@prisma/client";
```